### ISSUE TITLE
Doesn't work out of the box on newer versions of Postgresql

### ISSUE THREAD

**Issue by keenan-v1** [ajanata/PretendYoureXyzzy#232]
It would seem that your project is bound to Postgres 8 due to the JDBC drivers. I was having issues getting it to work with a straight install on Ubuntu 18.04 and found out it was due to the default postgres on that distro being Postgres 10.

To get it working I used Postgres 9.6 (docker) (since there doesn't seem to be a compatible 10 JDBC driver on Maven, though I didn't look too hard) and upgraded the JDBC driver to the latest.

The behavior on newer versions of Postgres is that it continuously opens connections and fails to communicate with the database until all connections are exhausted. The symptoms to look for are no card sets on a new game and a mysterious "timeout" error message in the chat. Also "View Cards" will fail with a stacktrace - thankfully it doesn't dump anything that can be used in an attack.

**Comment by keenan-v1** [comment]
Turns out just using Posgresql <10 works. I had another issue that was preventing things from working.

**Comment by Stary2001** [comment]
The official driver that's already being used - https://jdbc.postgresql.org/ - is available on Maven Central, but the 42.x version is under a different groupId of `org.postgresql` instead of `postgreql` for some reason. 
Changing the existing driver dependency to 
```
<dependency>
  <groupId>org.postgresql</groupId>
  <artifactId>postgresql</artifactId>
  <version>42.2.12</version>
</dependency>
```
appears to work fine so far for my instance.

**Comment by ajanata** [comment]
@Stary2001 mind opening a PR for that?

**Comment by Stary2001** [comment]
Done - #246

**Comment by ajanata** [comment]
Fixed in #246.
